src/schemas/json/fly.json

```diff
@@ -227,88 +265,337 @@
-                "description": "The HTTP method to be used for the check.",
+              {
                 "type": "string"
+              }
+            ]
+          },
+          "interval": {
+            "description": "Length of the pause between connectivity checks. Units are milliseconds unless you specify them like `10s` or `1m`.",
+            "oneOf": [
+              {
+                "type": "integer"
               },
-              "path": {
-                "description": "The path of the URL to be requested.",
+              {
                 "type": "string"
+              }
+            ]
+          },
+          "restart_limit": {
+            "default": 0,
+            "description": "The number of consecutive check failures to allow before attempting to restart the VM. The default is `0`, which disables restarts based on failed health checks.",
+            "type": "integer"
+          },
+          "timeout": {
+            "description": "The maximum time a connection can take before being reported as failing its healthcheck. Units are milliseconds unless you specify them like `10s` or `1m`.",
+            "oneOf": [
+              {
+                "type": "integer"
               },
-              "protocol": {
-                "description": "The protocol to be used (`http` or `https`)",
-                "type": "string",
-                "enum": ["http", "https"]
-              },
-              "tls_skip_verify": {
-                "type": "boolean",
-                "default": false,
-                "description": "When set to `true` (and `protocol` is set to `https`), skip verifying the certificates sent by the server."
-              },
-              "headers": {
-                "type": "object",
-                "description": "Set key/value pairs of HTTP headers to pass along with the check request."
+              {
+                "type": "string"
               }
-            }
+            ]
+          },
+          "method": {
+            "description": "The HTTP method to be used for the check.",
+            "type": "string"
+          },
+          "path": {
+            "description": "The path of the URL to be requested.",
+            "type": "string"
+          },
+          "protocol": {
+            "description": "The protocol to be used (`http` or `https`)",
+            "type": "string",
+            "enum": ["http", "https"]
+          },
+          "tls_skip_verify": {
+            "type": "boolean",
+            "default": false,
+            "description": "When set to `true` (and `protocol` is set to `https`), skip verifying the certificates sent by the server."
+          },
+          "headers": {
+            "type": "object",
+            "description": "Set key/value pairs of HTTP headers to pass along with the check request."
           }
         }
       }
     }
   },
   "description": "https://fly.io/docs/reference/configuration",
   "properties": {
+    "$schema": {
```

Comment:
I was wondering, if this fixes an error that was encountered, or it's more of a precaution.  I know that [Even Better TOML reads a comment directive](https://taplo.tamasfe.dev/configuration/directives.html) to find the schema, but I don't know if it looks at `$schema` or not.

Reply:
I set additionalProperties to false and VScode extension was showing an error without it, but maybe it's better to allow additionalProperties, i will remove it

Reply:
Thanks! Hm yeah, that's weird, maybe it has to do with the `fly.json` schema applying to a TOML file, which doesn't require the use of `$schema` properties (since it allows comments). The CONTRIBUTING recommends setting `additionalProperties` to `false`, but it kind of goes both ways. Setting it to `false` creates false positive errors when an application adds a field in a new release. And setting it to `true` decreases the chance that the schemas are updated when an application has a new release (since there are no errors), but conversely, users' won't get errors when one of their property fields has a typo.